### DISCUSSION TITLE
Make quickstart embeds automtaically generated based on snippets

### DISCUSSION THREAD

**Discussion by marcosnils** [dagger/dagger#12435] [maintainers]

**Comment by crjm** [comment]
@marcosnils do you have some context on this issue so we can work on it with @vikram-dagger?

**Comment by marcosnils** [comment]
@crjm I sent Vikram a shell script to automate part of this task. The ultimate goal would be to have a CI check that ensures that the snippets and the embed links are in sync. We can think together about what would be the best approach for this.
